homeassistant/components/netatmo.py

```diff
@@ -37,6 +38,7 @@
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Required(CONF_SECRET_KEY): cv.string,
         vol.Required(CONF_USERNAME): cv.string,
+        vol.Optional(CONF_DEVICES, default=DEFAULT_DEVICES): cv.ensure_list,
```

Comment:
I do not see a use case for making this configurable. It also should get validated against all valid values (which I guess are DEFAULT_DEVICES?)